WPM_T9.1.md

```diff
@@ -25,31 +25,31 @@ In diesem Zuge könnte man auch darüber nachdenken, ein solches Verfahren mit e
-Ein regelmäßiges, automatisches Matching und Merging soll die Qualität und Konsistenz der GND-Datensätze künftig erhöhen und auch die GND-Redakteur\*innen entlasten, indem die maschinell erkannten Dubletten unmittelbar als solche zur händischen Bearbeitung vorgeschlagen werden.
+Mit der Bereinigung von Dubletten verhält es sich ähnlich wie mit der Erschließung von Medien, dem Anlegen und Verknüpfen von Normdaten. Dubletten werden in der Regel dann händisch bearbeitet, wenn sie in der Erschließungsarbeit als Dubletten auffallen ("work in progress"). In der GND können Dubletten insbesondere dann gehäuft entstehen, wenn ein größeres Datenset von extern eingespielt wurde. Aus diesem Grund wird derzeit die sogenannte "GND-Toolbox" entwickelt ([Kett et al. 2019, S. 67 und passim](https://www.o-bib.de/article/view/5539/7880)).  Diese soll die Anbindung neuer Datenquellen aus nicht-bibliothekarischen Anwendungskontexten durch Bereitstellung von Schnittstellen unterstützen. Darüber hinaus soll die GND-Toolbox aber auch als Werkzeug zur Qualitätskontrolle der GND zur Verfügung stehen. Sowohl beim Pre-Processing von externen Datenbeständen als auch bei der Qualitätskontrolle kommen Match & Merge-Verfahren zum Einsatz. Im ersten Fall wird das externe Datenset gegen den Bestand der GND abgeglichen, um "Import-Kandidaten" zu identifizieren oder aber auszuschließen (siehe auch Prozess Nr. 4). Im zweiten Fall wird die GND gewissermaßen "gegen sich selbst gematched", um mögliche Dubletten zu identifizieren.
+Ein regelmäßiges, automatisches Matching und Merging soll die Qualität und Konsistenz der GND-Datensätze<!---COMMENT besser "GND-Sätze", denn in "GND" ist der Wortbestandteil "-Daten-" ja bereits enthalten ...--> künftig erhöhen und auch die GND-Redakteur\*innen entlasten, indem die maschinell erkannten Dubletten unmittelbar als solche zur händischen Bearbeitung vorgeschlagen werden.
```

Comment:
Tatsächlich würde ich es gerne bei "GND-Datensätze" belassen, auch wenn man im Alltag häufig von "Sätzen" ("Titelsätze"...) spricht. Schließlich steht das "D" in "GND" ja für die Datei als solche und nicht für "Daten". 🤓

Reply:
Das ist ein guter Einwand, den ich natürlich absolut gelten lasse!

Ich glaube, dann sind wir hier endgültig durch. Super!